### PR TITLE
BD-1023: Compliance Problems in hal-wifi-cfg80211

### PR DESCRIPTION
**Pull request by manigandanj**
Signed-off-by: Manigandan Gopalakrishnan <manigandan.gopalakrishnan@ltts.com>

### CODE REVIEW

File: source/wifi/client_wifi_hal.c [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include <fcntl.h>
```
Comment (mhughesacn) [inline source/wifi/client_wifi_hal.c:5]: Still missing header please

File: source/wifi/client_wifi_hal.h [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#ifndef __CLIENT_WIFI_HAL_H__
```
Comment (mhughesacn) [inline source/wifi/client_wifi_hal.h:1]: Missing header on new file

File: source/wifi/wifi_hal.c [hunk 3]
```diff
@@ -22,6 +22,14 @@
 * Licensed under the BSD-3 license
 */
 
+//Code in rxStatsInfo_callback, txStatsInfo_callback and in chanSurveyInfo_callback originates from:
```
Comment (mhughesacn) [inline source/wifi/wifi_hal.c:25]: Please use the text in BD-1023 on this line:
"Code in rxStatsInfo_callback and other callbacks is credited as follows:"

**Review by mhughesacn — CHANGES_REQUESTED** [review]
Hi Manigandan - please see my comments - thanks.

**Review by mhughesacn — APPROVED** [review]

**Comment by mhughesacn** [comment]
Hi Manigandan: Please will you make one more change which I noticed when I went back to the repo: Before the actual text of the ISC License at line 209, please will you put in a title as follows, and remove the Copyright for Johannes Berg?  (NOTICE is where the actual copyrights are given.)

ISC License
===========
Copyright YEAR AUTHOR

File: NOTICE [hunk 4]
```diff
@@ -10,6 +10,9 @@ within this component.
 Copyright 2014 Cisco Systems, Inc.
 Licensed under the Apache License, Version 2.0
 
+ISC License
```
Comment (mhughesacn) [inline NOTICE:13]: Sorry, crossed line here: please delete lines 13-15 and put them instead into the file called LICENSE before the text of the ISC license.  The LICENSE file is in the same directory as NOTICE.

File: LICENSE [hunk 5]
```diff
@@ -202,6 +202,9 @@
    limitations under the License.
 
 
+ISC License
+===========
+Copyright  YEAR  AUTHOR
 Copyright 2007 Johannes Berg
```
Comment (mhughesacn) [inline LICENSE:208]: Sorry Manigandan - want this line deleted too (line 208, Johannes Berg)

**Comment by mhughesacn** [comment]
Thanks Manigandan.  I cleared the blockages in the tool as well, and now it scans clean.

**Review by rajkumar154 — APPROVED** [review]

**Pull request merged**